timed/reports/management/commands/notify_supervisors_shorttime.py

```diff
@@ -90,7 +94,10 @@ def _get_supervisees_with_shorttime(self, start, end, ratio):
                     self._decimal_hours(reported),
                     self._decimal_hours(expected),
                     self._decimal_hours(balance),
-                    supervisee_ratio
+                    supervisee_ratio,
+                    self._decimal_hours(
+                        supervisee.calculate_worktime(start_year, today)[2]
+                    ),
```

Comment:
Depending on when the cronjob is run (morning, evening), this will yield different results.. I would prefer the balance of "yesterday" as it's more likely to be correct. If it's run in the morning, you will get a balance 8.5h below the actual balance, for example

Reply:
Oh and is worktime since the start of the year always correct? Shouldn't we take the employment into consideration as well?

Reply:
calculate_worktime already takes care of the employment length...  I will change end date to yesterday.

Reply:
Thought about it again. notify supervisor has an offset parameter which is set because it is expected that users may report their time too late. This needs to be taken into consideration as well for current balance. I changed the code accordingly.

Reply:
Maybe even better: calculate the balance at the end of the analyzed time period

Reply:
that's how I have done it now.